Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -234,6 +235,23 @@ def custom_callback(name, progress):
 image_series = itk.imread([image_series3d_filename, image_series3d_filename])
 assert image_series.GetImageDimension() == 3
 
+baseline_parameters = np.array([0.6563149 ,   0.58065837,  -0.48175367, -0.74079868, 0.37486398,  -0.55739959,  -0.14306664,   0.72271215, 0.67617978, -66., 69.,  32.])
```

Comment:
Are these (`0.6563149 , 0.58065837, -0.48175367,`...) arbitrary values?

Reply:
These are what are in the baseline input transform file.

Reply:
> These are what are in the baseline input transform file.

Thanks Matt, so those values are from Testing/Data/Input/LinearTransform.h5, right?

Reply:
Correct :+1:

Reply:
Ah, I see now, it's at https://data.kitware.com/#collection/57b5c9e58d777f126827f5a1/folder/57b76d6b8d777f10f269bc75

Reply:
Just double-checked the TransformParameters from LinearTransform.h5. According to the HDF Viewer: 

```
0.65631490118447
0.5806583745824385
-0.4817536741017158
-0.7407986817430222
0.37486398378429736
-0.5573995934598175
-0.14306664045479867
0.7227121458012518
0.676179776908723
-65.99999999999997
69.00000000000004
32.000000000000036

```

Which are _all close_ to the corresponding values of your `baseline_parameters`  👍